src/js/historical.cpp

```diff
@@ -69,7 +69,7 @@ namespace ccf::js
     JS_SetPropertyStr(
       ctx,
       js_receipt,
-      "nodeId",
+      "node_id",
```

Comment:
I'm surprised no (end-to-end?) test broke because of this?

Reply:
Yeah we don't have great coverage of the JS/TS API, especially the recent Receipt changes. I'd like to expand that, but in a separate PRs.

Reply:
(Of course another way of saying that is "this PR is completely untested, but _looks ok_"...)

Reply:
It may be worth tracking this in a new (or existing) GH issue?

Reply:
#3503

Reply:
I don't think the qjs coverage of receipts is any less than for C++, it's only v8 that's missing some bits. The node_id was once used before the PEM was in the receipts, to look up files, but isn't anymore.

It doesn't really make a lot of sense to have it _and_ the PEM, the thought was that we'd have full state historical queries, and make signatures more compact and do the lookup on receipt creation (or even leave it to the user), but it's not clear that this is the right tradeoff. We have a discussion scheduled about receipts, I think we'll want to cover that then.